source/OptimaJet.DWKit.StarterApplication/BackendServicesExtension.cs

```diff
@@ -55,7 +56,7 @@ public static IServiceCollection AddApiServices(this IServiceCollection services
             services.AddScoped<IEntityRepository<Notification>, NotificationRepository>();
             services.AddScoped<IEntityRepository<Product,Guid>, ProductRepository>();
             services.AddScoped<IEntityRepository<OrganizationStore>, OrganizationStoreRepository>();
-
+            services.AddScoped(typeof(IUpdateService<Project, int>), typeof(ProjectService));
```

Comment:
any reason this signature is different than the others?
```suggestion
            services.AddScoped<IUpdateService<Project, int>>, ProjectService>();
```

Reply:
for my own ocd reasons I would just standardize all of these to one syntax

Reply:
there isn't. probably got caught in using typeof with generics, and just didn't switch it back. thanks!